cegs_portal/search/views/v1/non_targeting_reos.py

```diff
@@ -92,3 +94,22 @@ def get_data(self, options, feature_id) -> Pageable[RegulatoryEffectObservation]
             non_targeting_reos.extend(list(reg_effects))
 
         return Paginator(non_targeting_reos, options["per_page"]).get_page(options["page"]), feature
+
+
+class DownloadTSVView(NonTargetRegEffectsView):
+
+    def get(self, options, *args, **kwargs):
```

Comment:
Won't this return only the first page of results?

Reply:
Hey, it returns all results.
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/45e5484f-6282-4a8e-bc66-447f90778427)

Reply:
I changed the default to 5 here. Cant I change it here?
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/30f30d8e-4195-4788-a2e2-f2994237a01e)

Reply:
Oh, I see, yes, you're right. 

I didn't look close enough and assumed you were using the data from the NonTargetRegEffectsView#get_data method, but you are using data you got from the `get` method.

Reply:
Yes! Ok seems like we are good to go here. Thanks Thomas!